### PR TITLE
Sergio/register script

### PR DESCRIPTION
**Pull request by olirods**
The mix script for registering users within the console. You can try it with `mix siwapp.register`

### CODE REVIEW

File: .dialyzer_ignore.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  {"lib/mix/tasks/siwapp.ex"},
+  {"lib/mix/tasks/siwapp.register.ex"}
+]
```
Comment (peillis) [inline .dialyzer_ignore.exs:4]: could this be a pattern for the whole directory? Something like "lib/mix/tasks/*"

**Review by peillis — COMMENTED** [review]
Nice job.

**Review by nurietzsche2021 — APPROVED** [review]
Excellent

File: .dialyzer_ignore.exs (hunk 1 above)
Comment (olirods) [reply .dialyzer_ignore.exs:3]: I've pushed a new commit with that. "lib/mix/tasks/*" didn't work, I was only able to do it with reg. exp: ~r/lib\/mix/

Comment (nurietzsche2021) [reply .dialyzer_ignore.exs:3]: Great choice, I think we found the correct solution. Hard to deal with RegEx in Elixir, right?

Comment (olirods) [reply .dialyzer_ignore.exs:3]: Haha yes!! Thank you for helping me :)

**Review by peillis — APPROVED** [review]

**Pull request merged**